src/docfx/build/page/BuildPage.cs

```diff
@@ -83,7 +74,39 @@ public static async Task<List<Error>> Build(Context context, Document file)
             return errors;
         }
 
-        private static async Task<(List<Error>, OutputMetadata)> GenerateOutputMetadata(
+        private static async Task<(List<Error> errors, object output, JObject metadata)> CreatePageOutput(Context context, Document file, JObject model)
```

Comment:
consider break it, this line is too long. I am looking for a tool to enforce max line with to be something like 140
```suggestion
        private static async Task<(List<Error> errors, object output, JObject metadata)>
            CreatePageOutput(Context context, Document file, JObject model)
```